Plutarch/Num.hs

```diff
@@ -42,6 +42,12 @@ class PNum (a :: PType) where
   default pfromInteger :: PNum (PInner a) => Integer -> Term s a
   pfromInteger x = punsafeDowncast $ pfromInteger x
 
+-- prohibit mixing arithmetic operators such as
+-- (2 #+ 3 #* 4) without explicit precedence.
+infix 6 #+
+infix 6 #-
+infix 6 #*
+
```

Comment:
Could this be a breaking change?

Reply:
I think this should already be in staging, no? Perhaps it got reverted. I gotta have a look.  It a change that I introduced.

Reply:
changed base branch, now this is resolved